### DISCUSSION TITLE
How do I use TouchID to unlock KeePassXC?

### DISCUSSION THREAD

**Discussion by albertinix** [keepassxreboot/keepassxc#7389] [Q&A] [answered]
I'm using KeePassXC on macOS BigSur. I would like to use TouchID to unlock the database, but I don't know how and haven't found a guide on this.

I tried checking the box when opening the app - but that doesn't seem to do anything for me.

Any tips?

**Comment by droidmonkey** [comment, marked as answer]
Check box, enter password. Lock database, ensure checkbox is still checked, just press OK and you will be prompted for touch id. This only persists for the period of time that keepassxc is running. If you log off or reboot you will lose quick unlock.  This process has been greatly improved by #7384

**Reply by albertinix** [reply]
Thank you, works perfectly.

**Reply by notDavid** [reply]
> This only persists for the period of time that keepassxc is running.

Are there any plans to make it persist regardless whether keepassxc has quit?

Something like the feature in Strongbox for iOS, where you can configure `Face ID Unlock` to `Require master password interval` = `app exit` / `n hours` / `n days` / `n weeks` / `never` ? (which of course requires storing the key in the OS keychain...)

**Reply by droidmonkey** [reply]
We are considering it, we already store the encrypted credentials in the key store. They just get erased on log off.

**Reply by notDavid** [reply]
Great, thank you @droidmonkey.

I assume i don't need to make a new issue/feature request for it since you're already considering it.

Or is there already an open issue for it which i missed?

**Reply by droidmonkey** [reply]
That is #7020
